agent/wptdriver/wpt_test.h

```diff
@@ -225,6 +225,11 @@ class WptTest {
   void      BuildScript();
   CAtlList<ScriptCommand> _script_commands;
 
+  //Timeout information
+  DWORD    _timeout_value_in_seconds;
+  bool     _enable_per_test_timeout;

```

Comment:
Don't see why enable_per_test_timeout is needed? Can't we just use the fact that timeout_value_in_seconds != 0 to mean that per test timeout is enabled?